lib/tasks/stateoscope.rake

```diff
@@ -1,8 +1,10 @@
 namespace :stateoscope do
   desc 'visualize state machine for a given class'
-  task :visualize, [:class, :state_machine_name] => :environment do |t, args|
-    fail ArgumentError, 'missing required argument <class>' unless args[:class].present?
+  task :visualize, %i[class state_machine_name current_state] => :environment do |_, args|
+    raise ArgumentError, 'missing required argument <class>' if args[:class].nil?
     klass = args[:class].classify.constantize
-    Stateoscope.visualize(klass, state_machine_name: args[:state_machine_name])
+    # class not needed in args
+    args = args.reject { |k, _| k == :class }.to_h
+    Stateoscope.visualize(klass, args)
```

Comment:
I believe this can be improved by using `Hash#delete`. It is more concise and we don't need to repeat the`:class` key:

    klass = args.delete(:class).classify.constantize
    Stateoscope.visualize(klass, args)

Reply:
yea I tried `Hash#delete` but didn't work. I actually just found out that it's not a `Hash` but a `Rake::TaskArguments`. I think it would actually be best to have `Stateoscope.visualize` just take one argument (a `Hash`), but I'm not gonna change that now.